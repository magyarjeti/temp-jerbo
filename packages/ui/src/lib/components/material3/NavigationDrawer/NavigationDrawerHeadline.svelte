<div class="typo-title-small py-3 pl-4 pr-6"><slot /></div>
