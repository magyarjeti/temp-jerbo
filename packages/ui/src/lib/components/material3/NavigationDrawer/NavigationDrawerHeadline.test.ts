import { render } from '@testing-library/svelte';
import { expect, test } from 'vitest';
import NavigationDrawerHeadline from './NavigationDrawerHeadline.svelte';

test('NavigationDrawerHeadline snapshot', () => {
	const { container } = render(NavigationDrawerHeadline);
	expect(container).toMatchSnapshot();
});
