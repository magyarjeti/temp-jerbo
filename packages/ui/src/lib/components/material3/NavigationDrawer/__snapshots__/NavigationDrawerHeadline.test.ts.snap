// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavigationDrawerHeadline snapshot 1`] = `
<div>
  <div
    class="typo-title-small py-3 pl-4 pr-6"
  />
</div>
`;
